src/app/bungie-api/destiny1-api.ts

```diff
@@ -26,7 +27,7 @@ export async function getCharacters(platform: DestinyAccount) {
       `/D1/Platform/Destiny/${platform.originalPlatformType}/Account/${platform.membershipId}/`
     )
   );
-  if (!response || Object.keys(response.Response).length === 0) {
+  if (!response || !hasKeys(response.Response)) {
```

Comment:
I might either prefer `_.isEmpty` from Lodash, or to replace all existing `_.isEmpty` with this helper. `_.isEmpty` is certainly not as simple so might not be what we want: https://lodash.com/docs/4.17.15#isEmpty

Reply:
oh yeah that's ubiquitous in dim. forgot it existed